test_app/scripts/entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+set -x
+
+sed -i '/^\[crb\]$/,/^enabled=0$/ s/enabled=0/enabled=1/' /etc/yum.repos.d/centos.repo
+dnf -y install \
+    python3.11 python3.11-pip python3.11-devel gcc openldap-devel \
+    xmlsec1 xmlsec1-openssl xmlsec1-devel libtool-ltdl-devel libpq-devel libpq postgresql
+if [[ ! -d /venv ]]; then
+    python3.11 -m venv /venv
+fi
+
+PIP=/venv/bin/pip
+PYTHON=/venv/bin/python3
+
+$PIP install -r requirements/requirements_all.txt
+$PIP install -r requirements/requirements_dev.txt
+
+echo "settings.DATABASE ..."
+$PYTHON manage.py shell -c 'from django.conf import settings; print(settings.DATABASES)'
+
+MAX_ATTEMPTS=10
+for i in $(seq 1 $MAX_ATTEMPTS); do
```

Comment:
This is doing a lot of what the bootstrap script does. Would it be best to just run the bootstrap script inside the container? So when you spin the container up, it pre-loads some data already to play with?

Reply:
The bootstrap script wants to do docker commands and munge a postgres container via the make target.